CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.9.0] - (Apr 14, 2020) ##
```

Comment:
@sebyx07 you need add this line:
```
[0.9.0]: https://github.com/rubycdp/ferrum/compare/v0.8.0...v0.9.0
```
[here](https://github.com/sebyx07/ferrum/blob/master/CHANGELOG.md#L190)  to make link works